mechafil/power.py

```diff
@@ -146,7 +146,13 @@ def forecast_power_stats(
     forecast_lenght: int,
     fil_plus_m: float = 10.0,
     duration_m: Callable = None,
+    qap_method: str = 'tunable'  # can be set to tunable or basic
```

Comment:
This option needs to be added to all the user functions that may want to change the default behaviour. From what I can see, this only includes the `run_simple_sim` function from the `sim.py` module. 

In addition, we should update the readme in order to explain what this parameter does.

Reply:
Thanks for the reminder - done.